x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/tool/CommandUtils.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.security.tool;
+
+import org.elasticsearch.core.Nullable;
+import java.security.SecureRandom;
+
+public class CommandUtils {
```

Comment:
I'm not fond of this class and its name but I was also not fond of copying `generatePassword` around in yet another place. Happy to get suggestions for alternatives

Reply:
I actually like it! I was trying to park this method somewhere too, but couldn't find a good place, so adding this Util class may be a good solution.